easybuild/easyconfigs/e/EMAN/EMAN-daily.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PackedBinary'
+
+name = 'EMAN'
+version = 'daily'
+
+homepage = 'http://www.msg.ucsf.edu/local/programs/eman'
+description = """EMAN is a powerful image processing library as well as a complete software suite 
+                for single particle reconstruction. This is a process for converting randomly 
+                oriented 2D projection images into a 3D model, and is typically used in conjunction 
+                with electron cryomicroscopy. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ncmi.bcm.edu/ncmi/software/eman/software_78/']
+sources = ['%(namelower)s.%(version)s.linux64-cluster.tar.gz']
+
+modextrapaths = {
+    'PYTHONPATH' : 'lib',
+}
+modextravars = {
+    'EMANDIR' : '$root',
+}
+
+sanity_check_paths = {
+    'files': ['bin/afmtomrc.py'],
+    'dirs': ['bin',
+             'include',
+             'lib',
+             'python'],
```

Comment:
these can go an a single line, no need to include `bin` here since you're already checking for a file in `bin`